prboom2/src/gl_sky.c

```diff
@@ -339,7 +339,7 @@ void gld_DrawSkyCaps(void)
   {
     if (dsda_MouseLook())
     {
-      gld_BindTexture(SkyBox.wall.gltexture, 0);
+      gld_BindSkyTexture(SkyBox.wall.gltexture);
```

Comment:
Any idea why it wasn't already like this?

Reply:
I think it was just an oversight in 1b9157a59c

Reply:
👍